taskt/UI/Forms/Supplement Forms/frmHTMLElementRecorder.cs

```diff
@@ -74,77 +65,75 @@ private void Document_Click(object sender, HtmlElementEventArgs e)
                 HtmlAgilityPack.HtmlDocument doc = new HtmlAgilityPack.HtmlDocument();
```

Comment:
Please cleanup `HtmlAgilityPack` namespace

Reply:
This one can't be shortened because it overlaps with another namespace

Reply:
That's fine.